content/source/docs/enterprise/admin/active-active.html.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: "enterprise"
+page_title: "Active/Active -  - Infrastructure Administration - Terraform Enterprise"
+---
+
+# Terraform Enterprise Active/Active
+
+When your organization requires increased reliability or performance from Terraform Enterprise that your current single application instance cannot provide, it is time to scale to the Active/Active architecture. Please refer to  [Terraform Enterprise Active/Active](https://www.terraform.io/docs/enterprise/install/active-active.html) for more information on moving to Active/Active.
+
+## Active/Active Admin Commands
+
+The Active/Active operational mode disables use of the Replicated Admin Console.  To accommodate this, there are now admin commands to facilitate configuration changes, safe application stops, and producing support bundles. There is a new container in the TFE architecture to support this called "tfe-admin". You must login to a node in the Active/Active cluster using SSH to run these commands from the command line.
+
+### Commands
+
+Note that `tfe-admin` is an alias for `replicated admin`, and can be used interchangeably.
+
+#### support-bundle
+
+```bash
+tfe-admin support-bundle
+```
+
+This command will generate a support bundle for all nodes. The support bundles will be uploaded to the same object store bucket that is used to store Terraform statefiles. The support bundles for a specific run of the admin command will all be uploaded to a directory with the same JobID (a timestamp in [RFC3339](https://tools.ietf.org/html/rfc3339) format). If you are sending a support bundle to HashiCorp Support, package up the associated bundles and send all in order to ensure all needed information is made available.
+
+Example upload structure
+
+```bash
+support-bundles
+└── 2020-11-10T02:03:05Z
+    ├── 10.0.0.5
+    │   └── replicated-support702524260.tar.gz
+    └── 10.0.0.6
+        └── replicated-support577188727.tar.gz
+```
+
+
+#### node-drain
+
+```bash
+tfe-admin node-drain 
+```
+
+This command will quiesce the current node and remove it from service. It will allow current work to complete and safely stop the node from picking up any new jobs from the Redis queue, allowing the application to be safely stopped. Currently, it only affects `localhost` (it does not support running on one node to drain other nodes). Currently, there is no reverse drain command - a restart is needed to restore the node.
+
+
+####  app-config
+
+```bash
+tfe-admin app-config -k <KEY> -v <VALUE>
+```
+
+This command allows users to make ad hoc/realtime application changes, such as `capacity_concurrency` via the CLI. Both an allowable `<KEY>` (setting name) and `<VALUE>` (new setting value) must be provided. A complete list of the current `app-config` settings can be found by running `replicatedctl app-config export`. You will be warned to restart the TFE application after running this command and must do so with a `replicatedctl app restart` command for the configuration changes to be in effect.
```

Comment:
They'll need top run `replicatedctl app restart` on each machine as well as that only runs locally. May need to investigate a way to do this via tfe-admin (though having it reboot the thing that runs it is a bit tricky).

Reply:
clarified